forgettable-frontend/src/components/EncounterCardSummary/EncounterSummary.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import Card from '@mui/material/Card';
+import {Avatar} from '@mui/material';
+import classes from './EncounterSummary.module.css';
+
+export default function EncounterSummary(props: { Name: any; Date: any; Description: any; Summary: any; }) {
+  const {
+    Name, Date, Description, Summary,
+  } = props;
+
+  return (
+    <div className={classes.Container}>
+      <Card
+        className={classes.Card}
+        sx={{maxWidth: 220, maxHeight: 240, bgcolor: 'var(--lcard)', borderRadius: 6, boxShadow: 0}}
+      >
+        <div className={classes.Header}>
+          <Avatar
+            alt={Name}
+            src="Avatar.png"
```

Comment:
Sorry, a couple more things: For this you'll need to pass in a img prop, then set src={props.img} so that we can display the person's custom image that they set